src/UrlHandlers/CrudUrlHandler.php

```diff
@@ -29,15 +29,19 @@ class CrudUrlHandler extends LeafRestUrlHandler
 
     private $leafClassStub;
 
-    public function __construct($modelName, $namespaceBase, $additionalPresenterClassNameMap = [], $childUrlHandlers = [])
+    public function __construct($modelName, $namespaceBase, $additionalPresenterClassNameMap = [], $childUrlHandlers = [], $prefix)
     {
         $namespaceBase = rtrim($namespaceBase, "\\");
         $this->namespaceBase = $namespaceBase;
 
         // Get the parent folder which will become our collection presenter
         $parts = explode("/", str_replace("\\", "/", $namespaceBase));
```

Comment:
Drop all three lines above into the else part to optimise performance (every little helps!)